documentation/reference/vespa-cmdline-tools.html

```diff
@@ -1499,11 +1500,141 @@ <h2 id="vespa-status-filedistribution">vespa-status-filedistribution</h2>
+      </td>
+    </tr><tr>
+      <th>--maxpendingsuperbuckets &lt;num&gt;</th>
+      <td>
+        Maximum pending visitor messages from the vespa-visit client.
+        If set, dynamic throttling of visitors is disabled <!-- ToDo: link to somewhere documenting throttling -->
+      </td>
+    </tr><tr>
+      <th>--maxtotalhits &lt;num&gt;</th>
+      <td>
+        Abort visiting when received this many total documents.
+        This is only an approximate number,
+        all pending work will be completed and those documents will also be returned
+      </td>
+    </tr><tr>
+      <th>-o, --timeout &lt;milliseconds&gt;</th>
+      <td>Time out visitor after given time</td>
+    </tr><tr>
+      <th>--ordering &lt;order&gt;</th>
+      <td>
+        Order to visit documents in.
+        Only makes sense in conjunction with a document selection involving id.order.
+        Values are <em>ascending</em> or <em>descending</em> <!-- ToDo: check. possibley fix document-select-language.html -->
+      </td>
+    </tr><tr>
+        <th>-p, --progress &lt;file&gt;</th>
+      <td>Use given file to track progress</td>
+    </tr><tr>
+      <th>--priority &lt;name&gt;</th>
+      <td>
+        Visitor <a href="services.html#clients">priority</a>. Defaults to NORMAL_3.
+        Use with care to avoid starving lower prioritized traffic in the cluster
+      </td>
+    </tr><tr>
+      <th>--processtime &lt;num&gt;</th>
+      <td>
+        Sleep this amount of millisecs before processing message.
+        (Debug option for pretending to be slow client)
+      </td>
+    </tr><tr>
+      <th>-r, --visitremoves</th>
+      <td>Include information of removed documents</td>
+    </tr><tr>
+      <th>-s, --selection &lt;selection&gt;</th>
+      <td><a href="document-select-language.html">Selection</a> string for which documents to visit</td>
+    </tr><tr>
+      <th>--skipbucketsonfatalerrors</th>
+      <td>Skip visiting super buckets with fatal error codes</td>
+    </tr><tr>
+      <th>--statistics &lt;args&gt;</th>
+      <td>Use CountVisitor for document statistics. Use comma-separated arguments</td>
+    </tr><tr>
+      <th>-t, --to &lt;timestamp&gt;</th>
+      <td>Only visit up to the given timestamp (microseconds)</td>
+    </tr><tr>
+      <th>--tracelevel &lt;level&gt;</th>
+      <td>Tracelevel ([0-9]), for debugging</td>
+    </tr><tr>
+      <th>-u, --buckettimeout &lt;milliseconds&gt;</th>
+      <td>Fail visitor if visiting a single bucket takes longer than this (default same as timeout)</td>
+    </tr><tr>
+      <th>-v, --verbose</th>
+      <td>Show progress and info on STDERR</td>
+    </tr><tr>
+      <th>--visitinconsistentbuckets</th>
+      <td>Don't wait for inconsistent buckets to become consistent</td>
+    </tr><tr>
+      <th>--visitlibrary &lt;string&gt;</th>
+      <td>Use the given visitor library</td>
+    </tr><tr>
+      <th>-x, --xmloutput</th>
+      <td>Output documents as XML</td> <!-- ToDo; deprecated? -->
```

Comment:
I believe it's deprecated but since it'll likely hang around until Vespa 8 we might as well keep it documented for now (but with a deprecation comment). @bratseth is all XML rendering deprecated entirely or do we expect to keep it around as a non-default format still?